site-manage/site-manage-group-section-role-helper/tool/src/java/org/sakaiproject/site/tool/helper/managegroupsectionrole/rsf/GroupImportStep2Producer.java

```diff
@@ -105,16 +105,17 @@ public void fillComponents(UIContainer tofill, ViewParameters viewParams, Compon
                 //check user is valid
                 String foundUserId = handler.lookupUser(userId);
                 if(foundUserId != null && handler.isValidSiteUser(foundUserId)){
+                    String userDisplayName = handler.getUserDisplayId(userId) + " ( " + userId + " )";
```

Comment:
Indent
The indentation here looks wrong, could you try to match it with the surrounding context, the issue maybe because of differences tabs and spaces.